src/control/lib/control/auto.go

```diff
@@ -30,7 +30,7 @@ const (
 	defaultTargetCount    = 16
 	defaultEngineLogFile  = "/tmp/daos_engine"
 	defaultControlLogFile = "/tmp/daos_server.log"
-	// NetDevAny matches any netdetect network device class
+	// NetDevAny matches any network device class
```

Comment:
Not your code but maybe this should be a constant in the hardware package?

Reply:
I think we went back and forth on this when it was first added, too. I'll move it, I think it's a little confusing here.